srcpkgs/samba4/template

```diff
@@ -0,0 +1,85 @@
+# Template file for 'samba4'
+pkgname=samba4
+version=4.9.4
+revision=1
+wrksrc="samba-$version"
+build_style=waf
+configure_args="--sysconfdir=/etc/samba --with-configdir=/etc/samba --localstatedir=/var --enable-fhs --with-lockdir=/var/cache/samba --with-piddir=/var/run/samba --with-logfilebase=/var/log/samba --without-pam --without-systemd --with-ads --with-shared-modules=idmap_ad,idmap_rid,idmap_adex,idmap_hash,idmap_tdb2,pdb_tdbsam,pdb_ldap,pdb_ads,pdb_smbpasswd,pdb_wbc_sam,pdb_samba4,auth_unix,auth_wbc,auth_server,auth_netlogond,auth_script,auth_samba4 --enable-cups --without-gettext"
```

Comment:
cut at 80 cols

Reply:
Why compile it without pam (`--without-pam`) but then quire it later?